### ISSUE TITLE
Avoid the `[AUTO] pre-commit update chore` by default

### ISSUE THREAD

**Issue by tmigot** [JuliaBesties/BestieTemplate.jl#503]
### Description

I used the "recommended" strategy for BestieTemplate, but receive "a lot" of email for pull request of such updates.
Would it make more sense to update these when updating to a new version of BestieTemplate instead?

### Can you help?

Sure!

**Comment by abelsiqueira** [comment]
I agree. I will change this to be an extra option and default to No.

**Comment by abelsiqueira** [comment]
This should be part of v0.17.0 when released
